docs/physics/gammaray_deposition.ipynb

```diff
@@ -0,0 +1,573 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from tardis.energy_input.base import GXPacket, main_gamma_ray_loop
</pre><p>at some stage we should rename <code>GXPacket</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1698/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>